### DISCUSSION TITLE
Team mode for managing quest density

### DISCUSSION THREAD

**Discussion by smichel17** [streetcomplete/StreetComplete#3204] [General]
Stray thought just now—

I realized that team mode is also useful if you're overwhelmed by the number of quests in an area. You could enable team mode with N people, and then you have 1/N as many quests displayed on the map! Not that I expect anyone new and overwhelmed to think of this :smile:

I might use this trick for pacing myself when using StreetComplete as motivation for going on walks— set the maximum group size (12), and then do the same walk 13 days in a row, as a different "member" of the group on each day (the last day, with team mode off, since there's some quests which will be missed using this method (the quests that depend on an answer from an earlier quest)

**Comment by matkoniecz** [comment]
Good idea, maybe I will also do this to reduce ratio of screen time to walking time.

> the quests that depend on an answer from an earlier quest

This can happen - but AFAIK only on way splitting. Filtering depends on element iD, so unlocked quest will always appear.

**Reply by smichel17** [reply]
I was mapping some apartments a few weeks ago where every building has 2 levels and they are arranged in repeated `C|` shapes (where the `|` is the road) around a central courtyard. With 6-10 buildings per `C`, I was almost standing still at some points (and it felt like cheating on the leaderboard :P).
